packages/api/src/controllers/farmExpenseTypeController.js

```diff
@@ -25,9 +25,28 @@ const farmExpenseTypeController = {
       try {
         const data = req.body;
         data.expense_translation_key = data.expense_name;
```

Comment:
I think we will need to refactor this to handle the translation key snake case formatting